src/sagemaker/workflow/airflow.py

```diff
@@ -269,3 +273,117 @@ def tuning_config(tuner, inputs, job_name=None):
         tune_config['S3Operations'] = s3_operations
 
     return tune_config
+
+
+def prepare_framework_container_def(model, instance_type, s3_operations):
+    """Prepare the framework model container information. Specify related s3 operations for Airflow to perform.
+    (Upload source_dir)
```

Comment:
nitpicks: s/s3/S3 and backticks around `source_dir`. same below

Reply:
Updated.

Reply:
don't forget about making "source_dir" into \``source_dir\``